jupyterhub/templates/hub/rbac.yaml

```
@@ -3,6 +3,10 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ include "jupyterhub.hub.fullname" . }}
+  annotations:
+    {{- with .Values.rbac.serviceAccount.annotations }}
+    {{- . | toYaml | nindent 4 }}
+    {{- end }}
```

This would render to...

```
annotations:
labels:
  # ...
```

If rbac.serviceAccount.annotations is falsy, and that would be invalid k8s YAML specification. This on the other hand would be okay.

```suggestion
  {{- with .Values.rbac.serviceAccount.annotations }}
  annotations:
    {{- . | toYaml | nindent 4 }}
  {{- end }}
```